### ISSUE TITLE
Question: How to match From: or Subject: header?

### ISSUE THREAD

**Issue by murty2** [OpenSMTPD/OpenSMTPD#1136]
Is it possible in OpenSMTPD to match an incoming message "**From:**" or "**Subject:**" header and then reject email if it matches?

If not, is it possible to to that using a filter in phase commit?

**Comment by idfake** [comment]
No, it's not possible. You would need to write a custom filter acting on DATA.
Alas there's filter-rspamd, mabe rspamd can be configured to to what you want.
